playground/src/pages/CodingEnvironment.tsx

```diff
@@ -142,19 +142,12 @@ export default function CodingEnvironment(): JSX.Element {
         const start = Date.now();
         const { stack_output, trace_len, overflow_addrs, proof }: Outputs =
           prove_program(code, inputs);
-        const overflow = overflow_addrs ? overflow_addrs.toString() : null;
-        if (overflow) {
-          setOutput(`{
+        const overflow = overflow_addrs ? overflow_addrs.toString() : "[]";
```

Comment:
nit:

```suggestion
        const overflow = overflow_addrs.toString();
```

or 

```suggestion
        const overflow = overflow_addrs ? overflow_addrs.toString() : "";
```

as it is, we would have the output set to `[[]]`, which is probably not the intention. (I haven't check what type is `overflow_addrs` to approach is best)